ovirt-ansible-collection.spec.in

```diff
@@ -11,6 +11,12 @@ License:	ASL 2.0 and GPLv3+
 BuildArch:	noarch
 Url:		http://www.ovirt.org
 
+BuildRequires:	ansible-core
+BuildRequires:	ansible-test
+%if 0%{?rhel} > 7 && 0%{?rhel} < 9
+BuildRequires:	glibc-langpack-en
```

Comment:
What do we need this for?

Reply:
The ansible-test fails with `/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)` without it

Reply:
OK